Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM articulate/articulate-node:8-jessie
+FROM articulate/articulate-node:8
 ARG vips_version=8.5.6
 ENV LD_LIBRARY_PATH=/usr/local/lib
 
 RUN echo 'deb http://httpredir.debian.org/debian stretch main' >> /etc/apt/sources.list && \
-    echo 'APT::Default-Release "jessie";' > /etc/apt/apt.conf.d/10apt && \
+    echo 'APT::Default-Release "stretch";' > /etc/apt/apt.conf.d/10apt && \
```

Comment:
This seems weird, is this needed now that we're on stretch?

Reply:
This may have breaking changes to our image crushing libs, and will require a bit of testing before we can merge.